saleor/product/models.py

```diff
@@ -454,6 +453,7 @@ class Collection(SeoModel):
     background_image = VersatileImageField(
         upload_to='collection-backgrounds', blank=True, null=True)
     is_published = models.BooleanField(default=False)
+    description = models.TextField(blank=True, null=True)
```

Comment:
In the Category model, this field is not nullable (while it allows for blank values). Can we keep consistency and have the same signature here? This affects `schema.graphql` so we'd have to regenerate it. For migrations, we could just change the `0078_auto_20181119_0535.py` accordingly.